src/ui/components/Difference/index.jsx

```diff
@@ -126,6 +157,16 @@ function Difference() {
     };
   }, []);
 
+  useEffect(() => {
+    if (isPaginationClicked.type === "prev") {
+      postMessage("PREV_DIFFERENCE_RECTANGLE");
+    }
+
+    if (isPaginationClicked.type === "next") {
+      postMessage("NEXT_DIFFERENCE_RECTANGLE");
+    }
+  }, [isPaginationClicked]);
```

Comment:
- `P2`
```suggestion
  useEffect(() => {
    if (isPaginationClicked.type === "prev") {
      postMessage("PREV_DIFFERENCE_RECTANGLE");
      
      return;
    }

    postMessage("NEXT_DIFFERENCE_RECTANGLE");
  }, [isPaginationClicked]);
```

`prev`와 `next`만 있기 때문에 `if문`을 두개 써줄 필요가 없을 것 같습니다!